cypress/features/regression/dialogFullScreen.feature

```diff
@@ -86,4 +86,19 @@ Feature: Dialog Full Screen component
       And clear all actions in Actions Tab
       And I open component preview
     When I click closeIcon in IFrame
-    Then cancel action was called in Actions Tab
\ No newline at end of file
+    Then cancel action was called in Actions Tab
+
+  @positive
+  Scenario: Verify that most top dialog is closed by pressing Esc key
+    Given I open "Dialog Full Screen" component page "with-nested-dialog"
+      And I open component preview
+      And I wait 500
+      And I open component preview using "Open Nested Dialog" text
+      And Dialog is visible in IFrame
```

Comment:
```suggestion
      And Nested Dialog is visible in IFrame
```

Reply:
it is not need to do this change - it will use the same locator inside this new step.